src/parse/internal_package.go

```diff
@@ -10,39 +11,12 @@ import (
 
 const InternalPackageName = "_please"
 
-// TODO(jpoole): Make this the magic bindata thing once go 1.16 is out
 // TODO(jpoole): make langserver configurable
-const internalPackageTemplateStr = `
-remote_file(
-    name = "download",
-    url = f"{{ .DownloadLocation }}/{{ .OS }}_{{ .Arch }}/{{ .PLZVersion }}/please_tools_{{ .PLZVersion }}.tar.xz",
-)
-
-# Can't use extract = True, as that would depend on :jarcat
-genrule(
-  name = "please_tools",
-  srcs = [":download"],
-  cmd = "tar -xf $SRC",
-  outs = ["please_tools"],
-)
-
-{{ range $tool := .Tools }}
-genrule(
-    name = "{{$tool}}".removesuffix(".jar"),
-    cmd = f"mv $SRC/{{$tool}} $OUT",
-    srcs = [":please_tools"],
-    outs = ["{{$tool}}"],
-    visibility = ["PUBLIC"],
-    binary = True,
-)
-{{ end }}
-
-`
-
-var internalPackageTemplate = template.New("_please")
+//go:embed internal.tmpl
+var internalPackageTemplateStr string
 
 func GetInternalPackage(config *core.Configuration) (string, error) {
-	t, err := internalPackageTemplate.Parse(internalPackageTemplateStr)
+	t, err := template.New("_please").Parse(internalPackageTemplateStr)
```

Comment:
Were we calling parse concurrently on this template or something?

Reply:
Yeah it seems so - I assume it can happen if we have a subrepo with an internal package too. Not sure how sensible a thing that is but this is an easy enough fix for now.